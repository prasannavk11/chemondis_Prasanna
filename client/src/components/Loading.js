import React from "react";

const Loading = React.memo(() => {
  return (
    <div className="d-flex justify-content-center">
      <div className="spinner-border" role="status"></div>
    </div>
  );
});

export default Loading;
