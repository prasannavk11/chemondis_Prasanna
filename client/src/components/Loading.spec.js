import { render } from "@testing-library/react";
import Loading from "./Loading";

afterEach(() => {
  jest.clearAllMocks();
  jest.resetAllMocks();
});

describe("Loading", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<Loading />);
    expect(baseElement).toMatchSnapshot();
  });
});
