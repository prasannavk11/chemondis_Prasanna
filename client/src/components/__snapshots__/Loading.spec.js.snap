// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render successfully 1`] = `
<body>
  <div>
    <div
      class="d-flex justify-content-center"
    >
      <div
        class="spinner-border"
        role="status"
      />
    </div>
  </div>
</body>
`;
